macros/GroupData.json

```diff
@@ -101,41 +101,28 @@
             "events":     [ "message" ]
         },
         "Browser API": {
-            "guides":     [ { "url":   "/docs/Web/API/Using_the_Browser_API",
+            "guides":     [ { "url":   "/docs/Mozilla/Gecko/Chrome/API/Browser_API/Using",
```

Comment:
I moved this to [`"/docs/Mozilla/Gecko/Chrome/API/Browser_API/Using_the_Browser_API"`](https://developer.mozilla.org/docs/Mozilla/Gecko/Chrome/API/Browser_API/Using_the_Browser_API)

Reply:
Well, I didn't really want it moved there. I felt the amount of words in the slug was redundant, so I was trying something else.